common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -78,11 +78,8 @@
 %></%def>
 
 <%def name="include(path)"><%
-from django.conf import settings
-from django.template.engine import Engine
-from django.template.loaders.filesystem import Loader
-engine = Engine(dirs=settings.DEFAULT_TEMPLATE_ENGINE['DIRS'])
-source, template_path = Loader(engine).load_template_source(path)
+from django.template.loader import get_template
+source = get_template(path)
```

Comment:
The only reason to load files through Mako is if they are dynamically updated per request, but that makes for very inefficient code that can't be cached. The benefit of using the ``text!`` plugin is that the files don't need to get loaded by Mako. In particular, in production they are bundled using RequireJS Optimizer into the root file financial_assistance_form_factory.js.